### PR TITLE
Add google_analytics_id to Sufia configuration

### LINKED ISSUE #2295: Where do we put google_analytics_id?
It looks like this was removed from Sufia.config, but there are still multiple references:
https://github.com/projecthydra/sufia/blob/43a3271050de6db0bec82847882bf9d88640e3ad/app/views/_ga.html.erb#L3
https://github.com/projecthydra/sufia/blob/c3e0ffda0abe9b1bd8fadc8d8d4323eb3201c3af/app/services/sufia/analytics.rb#L62

Do we follow CurationConerns?

https://github.com/projecthydra/curation_concerns/blob/a1fdf2d2deedf089f2ba52dc8a03208f26e0b344/config/locales/curation_concerns.en.yml#L13

### PR DESCRIPTION
**Pull request by awead**
Fixes #2295 

This was removed in CurationConcerns, https://github.com/projecthydra/curation_concerns/pull/867

@projecthydra/sufia-code-reviewers

### CODE REVIEW

**Comment by mjgiarlo** [comment]
The fact that our test suite passes without this config change makes me wonder if this PR might be an opportunity to add a new test that would fail without the new `attr_writer`. Otherwise 👍

**Comment by awead** [comment]
@mjgiarlo I've added tests for all config options.

File: spec/test_app_templates/lib/generators/test_app_generator.rb [hunk 1]
```diff
@@ -57,4 +57,10 @@ def enable_arkivo_api
   def relax_routing_constraint
     gsub_file 'config/initializers/arkivo_constraint.rb', 'false', 'true'
   end
+
+  # Turbolinks 5.0 is causing tests to fail with PhantomJS 2.1.1
+  def restrict_turbolinks_version
```
Comment (mjgiarlo) [inline spec/test_app_templates/lib/generators/test_app_generator.rb]: You can add this to `Gemfile.extra`

**Comment by mjgiarlo** [comment]
:red_circle:

File: spec/test_app_templates/lib/generators/test_app_generator.rb (hunk 1 above)
Comment (awead) [inline spec/test_app_templates/lib/generators/test_app_generator.rb]: No, because it gets added automatically when Rails is generated.

**Comment by awead** [comment]
@mjgiarlo yeah, I don't know how to make this work, other than removing PhantomJS from Travis, but that's not a viable solution.

File: spec/test_app_templates/lib/generators/test_app_generator.rb (hunk 1 above)
Comment (mjgiarlo) [inline spec/test_app_templates/lib/generators/test_app_generator.rb]: :ghost: Dang.

**Comment by mjgiarlo** [comment]
The error I'm seeing currently is:

```
You have requested:
  coffee-rails ~> 4.1.0
```

Is that related to the phantom thing?

**Pull request closed without merging**